README.md

```diff
@@ -22,6 +22,8 @@ Removes old snapshots.  Retention criteria are:
 
 ## Installation
 
```

Comment:
I think it would be worthwhile to callout near the top of the README that some things here are specific to LSST DM, such as the `.gitconfig`.